maat-orchestration/src/test/java/uk/gov/justice/laa/crime/orchestration/service/FeatureDecisionServiceTest.java

```diff
@@ -118,6 +119,40 @@ void givenUserHasFeatureToggleWithTheCorrectAction_whenFeatureToggleMethodIsInvo
         Assertions.assertTrue(result);
     }
 
+    @ParameterizedTest
+    @MethodSource("userHasFeatureToggleWithRequiredAction")
```

Comment:
Instead of adding a new test that largely duplicates the existing one with just one line changed, it would be cleaner and more maintainable to update the `userHasFeatureToggleWithRequiredAction` method source.

Reply:
Updated.